odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/AlertRepositoryImpl.java

```diff
@@ -119,6 +125,68 @@
             .collect(Collectors.toList());
     }
 
+    @Override
+    public Page<AlertDto> listDependentObjectsAlerts(final int page, final int size, final long ownerId) {
+        final List<String> ownOddrns = getObjectsOddrnsByOwner(ownerId);
+
+        final CommonTableExpression<Record1<String>> cte = getChildOddrnsLinageByOwnOddrnsCte(ownOddrns);
+
+        final List<Field<?>> selectFields = Stream
+            .of(ALERT.fields(),
+                DATA_ENTITY.fields(),
+                DATA_ENTITY_SUBTYPE.fields())
+            .flatMap(Arrays::stream)
+            .collect(Collectors.toList());
+
+        final List<AlertDto> data = dslContext.with(cte)
+            .select(selectFields)
+            .select(jsonArrayAgg(field(DATA_ENTITY_TYPE.asterisk().toString())).as(AGG_TYPES_FIELD))
+            .from(DATA_ENTITY)
+            .join(cte.getName()).on(field(cte.getName() + ".child_oddrn", String.class).eq(DATA_ENTITY.ODDRN))
+            .join(ALERT).on(DATA_ENTITY.ODDRN.eq(ALERT.DATA_ENTITY_ODDRN))
+            .join(TYPE_ENTITY_RELATION).on(TYPE_ENTITY_RELATION.DATA_ENTITY_ID.eq(DATA_ENTITY.ID))
+            .join(DATA_ENTITY_TYPE).on(DATA_ENTITY_TYPE.ID.eq(TYPE_ENTITY_RELATION.DATA_ENTITY_TYPE_ID))
+            .join(DATA_ENTITY_SUBTYPE).on(DATA_ENTITY_SUBTYPE.ID.eq(DATA_ENTITY.SUBTYPE_ID))
+            .where(DATA_ENTITY.ODDRN.notIn(ownOddrns))
+            .groupBy(selectFields)
+            .fetchStream()
+            .map(this::mapRecord)
+            .collect(Collectors.toList());
+
+        return Page.<AlertDto>builder()
+            .data(data)
+            .hasNext(true)
```

Comment:
Why do we always set hasNext as true and total as 0? @DementevNikita i see, that it copy-pasted from listByOwner method, is it expected?